app/pods/publications/publication/proofs/documents/template.hbs

```diff
@@ -195,17 +210,17 @@
 {{#if this.isPieceUploadModalOpen}}
   <Publications::Publication::Proofs::ProofUploadModal
     @publicationFlow={{this.publicationFlow}}
-    @isCorrected={{true}}
-    @isReceived={{false}}
-    @onSave={{this.saveCorrectionDocument}}
+    @isSource={{this.isPieceUploadSource}}
+    @isCorrected={{this.isPieceUploadCorrected}}
```

Comment:
`isSource` wordt niet gebruikt in proof-Upload-Modal (enkel in de component argument lijst)
Mag dus hier weg en ook in die component `proof-Upload-Modal.js`